### DISCUSSION TITLE
Payments not showing in stripe. Help!!!

### DISCUSSION THREAD

**Discussion by octxxiactual** [Glench/ExtPay#244] [Q&A]
I have noticed new users and even made two payments myself to verify everything works. The payments went through but nothing reflecting in my stripe after 3 weeks.

**Comment by burkybang** [comment]
I don't believe @Glench is available anymore. He has not been heard from in over 2 months.

**Comment by octxxiactual** [comment]
Who can I talk to.

**Reply by burkybang** [reply]
He was the only contact I know about 🤷‍♂️

**Comment by octxxiactual** [comment]
Thanks, I will shoot him and email see if I can reach him that way.

**Reply by burkybang** [reply]
That's what I did for the past 2 months.

**Comment by octxxiactual** [comment]
Ah, I see.

**Comment by Glench** [comment]
Yes, just email me from the email you signed up to Extensionpay.com with. I can't take a look at your account just using your Github username.

And as I said in other threads, I had an extended health issue but ExtensionPay has still been working just fine and making developers money the whole time :)
